docs/modules/ROOT/pages/whats-new.adoc

```diff
@@ -8,3 +8,29 @@ Below are the highlights of the release, or you can view https://github.com/spri
 
 * Added javadoc:org.springframework.security.web.authentication.preauth.x509.SubjectX500PrincipalExtractor[]
 * Added OAuth2 Support for xref:features/integrations/rest/http-interface.adoc[HTTP Interface Integration]
+
+== OAuth 2.0
+
+=== Username Expression Support for Nested Attributes - https://github.com/spring-projects/spring-security/pull/16390[gh-16390]
+
+OAuth2 Client now supports SpEL expressions for extracting usernames from nested UserInfo responses, eliminating the need for custom `OAuth2UserService` implementations in many cases. This is particularly useful for APIs like Twitter API v2 that return nested user data:
+
+[source,yaml]
```

Comment:
The yaml won't work yet because Spring Boot has not added support. Please remove it for now. If you like, you can save it in another branch in its own commit and we can use it when Spring Boot adds the support.